napari/_qt/widgets/qt_dims_slider.py

```diff
@@ -292,6 +292,7 @@ def loop_mode(self, value):
                 reversing direction when the maximum or minimum frame
                 has been reached.
         """
+        assert value in LoopMode.keys()
```

Comment:
could also use `value = LoopMode(value)`, in which case you'd get:
```
ValueError: 'bad value' is not a valid LoopMode
```

Reply:
Sure. 

```suggestion
        value = LoopMode(value)
```